packages/gatsby/src/bootstrap/load-plugins/validate.ts

```diff
@@ -197,16 +197,23 @@ export async function validatePluginOptions({
           }
 
           try {
-            // All plugins have "plugins: []"" added to their options in load.ts, even if they
-            // do not have subplugins. We add plugins to the schema if it does not exist already
-            // to make sure they pass validation.
+            if (typeof plugin.pluginOptions === `undefined`) {
+              return null
+            }
+
             if (!optionsSchema.describe().keys.plugins) {
```

Comment:
according to TS the [`IPluginInfoOptions`](https://github.com/gatsbyjs/gatsby/blob/3e9279563f76cef4e6a25ccb52aef85b5f6c2b6d/packages/gatsby/src/bootstrap/load-plugins/types.ts#L26) type (the correct one for out `pluginOptions`) can also have a `path` key, do we need to add that as well here?

Reply:
I have no clue but great catch! cc @wardpeet

Reply:
path & pathExists are part of the create-pages plugin. Only the plugins key is always there.